apps/sms/views/shared/js/utils.js

```diff
@@ -963,6 +963,17 @@
       });
 
       return defer.promise;
+    },
+
+    /**
+     * Check if device have more than 1 active SIM.
+     * @returns {boolean} True if the device has more than 1 SIM port and at
+     *  least 2 SIMs are inserted and false if device only has 1 SIM port or
+     *  mozIccManager not supported.
+     */
+    hasSeveralSim() {
```

Comment:
IIRC the remaining `Settings.hasSeveralSim` should be in the services originally,  and it's not working to import `Utils` in the worker because of API. So I'm thinking of moving the `hasSeveralSim` from service to the client, and there would be no other API dependency inside the messaging service. wdyt?